.github/workflows/build-release-packages.yml

```
@@ -0,0 +1,86 @@
+name: Build Release Packages
+
+on:
+  release:
+    types: [published]
+  
+  workflow_dispatch:
+    inputs:
+      version:
+        description: Version of Valkey to build
+        required: true
+
+jobs:
+  # This job provides the version metadata from the tag for the other jobs to use.
+  release-build-get-meta:
+    name: Get metadata to build
+    runs-on: ubuntu-latest
+    outputs:
+      version: ${{ steps.get_version.outputs.VERSION }}
+    steps:
+
+      - run: |
+          echo "Version: ${{ inputs.version || github.ref_name }}"
+        shell: bash
+
+      # This step is to consolidate the three different triggers into a single "version"
+      # 1. If manual dispatch - use the version provided.
+      # 3. If tag trigger, use that tag.
+      - name: Get the version
+        id: get_version
+        run: |
+            VERSION="${INPUT_VERSION}"
+            if [ -z "${VERSION}" ]; then
+              exit 1
+            fi
+            echo "VERSION=$VERSION" >> $GITHUB_OUTPUT
```

Indentation is still all over the place.
```suggestion
          VERSION="${INPUT_VERSION}"
          if [ -z "${VERSION}" ]; then
            exit 1
          fi
          echo "VERSION=$VERSION" >> $GITHUB_OUTPUT
```